test/test.py

```diff
@@ -50,6 +50,16 @@ def test_partialname(self) -> None:
         assert indi
         self.assertTrue("None" not in indi.getLabel())
 
+    def test_nosex(self) -> None:
+        # if there is no sex, this should fail and indicate line number
+        configDict = {
```

Comment:
If this is a work in progress test, then disable it, so `make check` keeps passing after accepting this PR.